extensions/vscode-api-tests/src/singlefolder-tests/terminal.shellIntegration.test.ts

```diff
@@ -87,6 +87,16 @@ import { assertNoRpc } from '../utils';
 		await closeTerminalAsync(terminal);
 	});
 
+	if (platform() === 'darwin') {
+		test('Test if env is set', async () => {
+			const { shellIntegration } = await createTerminalAndWaitForShellIntegration();
+			const env = shellIntegration.env;
+			ok(env);
+			ok(env['shellPath']);
+			strictEqual(env['shellPath'], '/bin/bash');
```

Comment:
I don't think `shellPath` would be on the env would it? For this I'd just test that env['PATH'] exists and has a length of > 0

Reply:
changes right here: https://github.com/microsoft/vscode/pull/237602/commits/5560bdc72cb3d1d95643d31706d660f0abc21859